routes/index.js

```diff
@@ -14,10 +15,17 @@ router.get('/download/:email/:uniqueId', function (req, res, next) {
 });
 
 router.get("/github", function (req, res, next) {
-  req.session.uniqueId = req.query.uniqueId;
-  req.session.email = req.query.email
-  req.session.gitURL = req.query.gitURL
-  next()
+  if (validation.isGitSSH(req.query.gitURL) || validation.isGitHTTPS(req.query.gitURL)) {
```

Comment:
now it is not working. but we can make it work by changing the deploy generator code a bit.I'll raise a separate issue for this.